simple-service-webapp-tests/src/test/java/com/example/TestTest.java

```diff
@@ -112,58 +112,33 @@ public void testPutMethod() throws IOException,ParseException {
     }
 
     @Test
-    public void getLoginToken() throws IOException,ParseException {
```

Comment:
Also JUnit test classes should be focused on testing a single unit, for example, let's use rename this TestTestClass and call it UserStoriesRestTest, use the BeforeClass annotated method to set the token, then make simpl crud tests for user stories. These test should contain asserts, so testing create would mean adding a new user story, then fetching the created user story by ID, and asserting that it exitst